CAIPs/caip-275.md

```diff
@@ -199,91 +202,82 @@ However, this configuration is not recommended as having multiple wallets genera
-3. Application uses its configured name resolvers to resolves the name and its linked authentication flows under the `authenticator` text record.
+3. Application uses its configured name resolvers to resolves the name authentication flows under the `authenticator` text record.
    If the authenticator text record is a URL, client application sends an HTTP GET request to it in order to obtain users authentication flows.
    This is not needed if the authenticator text record already has a stringified JSON in it.
-4. (Optional) Validate the response from name resolver is for the same address resolved by the Crypto Domain NFT.
-5. Application filters invalid authentication flows and initiates the one that can be completed which could be by opening a new window in the user's browser to the authenticator URL, triggering the browser wallet, etc.
-   If it is not supported due to platform or some other requirement, it can continue with the next one
-6. If no authentication flow can be processed by the application, then display this situation accordingly to the user.
-7. Wallet, when needed, will trigger the user for authentication and connection approval before stablishing the connection with the wallet.
-8. After wallet approves connection, application can match the domain name, the user's wallet address, and an authenticated session to the ones requested previously to validate user is not connecting with another wallet.
-9. When everything succeeds, connection should be stablished between application and the wallet found and triggered under the name provided by the user.
+4. Application filters invalid authentication flows and initiates the one that can be completed which could be by opening a new window in the user's browser to the authenticator URL, triggering the browser wallet, etc.
+   If it is not supported due to platform or some other requirement, it can continue with the next one.
+5. If no authentication flow can be processed by the application, then display this situation accordingly to the user.
+6. Wallet, when needed, will trigger the user for authentication and connection approval before establishing the connection with the wallet.
+7. After wallet approves connection, application can match the domain name, the user's wallet address, and an authenticated session to the ones requested previously to validate user is not connecting with another wallet.
+8. When everything succeeds, connection should be established between application and the wallet found and triggered under the name provided by the user.
 
 ## Connection properties
 
 How the dApp actually requests signatures and talks to the signer will vary depending on the authenticator and the chain being used.
-One option for session management is WalletConnect, where the "authenticated session" returned is actually a WalletConnect session.
+One option for session management is [Wallet Connect][], where the "authenticated session" returned is actually a WalletConnect session.
```

Comment:
```suggestion
One option for session management is to create a session with the [CAIP-25][] protocol, where the "authenticated session" token returned is actually a JSON-RPC session token, such as that used by the [Wallet Connect] JSON-RPC relay network.
```

There's other CAIP-25 implementations in the works so important to generalize the terminology here!